src/dmangle.d

```diff
@@ -88,6 +94,9 @@ private immutable char[TMAX] mangleChar =
     //              X   // variadic T t...)
     //              Y   // variadic T t,...)
     //              Z   // not variadic, end of parameters
+    //              %   // Type backward reference
+    //              ?   // symbol backward reference
+    //              $   // identifier backward reference
```

Comment:
If trusting bison to check the grammar is good enough, it seems that using a prefix character in LName is ok if the character is different from any type encoding.